src/app/views/data/data.js

```diff
@@ -20,6 +20,9 @@ class DataViewController extends ADBMapController {
     this.$log.debug('soum code:', this.soumCode);
 
     this.soumData.geojson(this.soumCode).then(geojson => this._addGeoJSONLayer(geojson));
+    this.soumData.load(this.soumCode).then(soum => {
```

Comment:
I'm not sure how to do it in one line and keep my linter happy without creating a named function just for an assignment. Would it make sense to combine `soumData.geojson` into `soumData.load` and have the callback handle both actions at once?